CHANGES.rst

```diff
@@ -12,6 +12,11 @@ esa/xmm-newton
 Service fixes and enhancements
 ------------------------------
 
+casda
+^^^^^
+
+- Add ability to stage and download ASKAP data. [#1706]
```

Comment:
this seems to be a significant addition, so I may move it up to new services at the time of tagged release